operations/network/management_network/canu_install_update.md

```diff
@@ -0,0 +1,43 @@
+# Upgrade Canu
+
+## Prerequisite
+
+Before using the CSM Automated Network Utility (CANU) to test, validate, or configure the network, ensure that CANU is running on the latest version. 
```

Comment:
```suggestion
Before using the CSM Automated Network Utility (CANU) to test, validate, or configure the network, ensure that CANU is running on the latest version. 

CANU can be run from your personal workstation (the instructions below are targeted at Mac users), or on the Shasta NCN nodes.

Since CANU is a python application, it can be run on Linux and Mac, but the RPM is not currently designed to support multiple operating system environments.  The CANU project can be cloned from Github and run directly by Python, but the project dependencies will need to be installed manually.  This process is not supported by this documentation.

The Windows operating system is untested and currently not officially supported.  We recommend that Windows users install or update CANU on the Shasta NCN nodes instead of attempting a workstation installation.


```